<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RigidBodyComp</class>
 <widget class="ComponentGroup&lt;Raz::RigidBody&gt;" name="RigidBodyComp">
  <property name="windowTitle">
   <string>Rigid body</string>
  </property>
  <property name="title">
   <string>Rigid body</string>
  </property>
  <layout class="QFormLayout" name="rigidBodyLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="massLabel">
     <property name="text">
      <string>Mass</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="ValuePicker" name="mass">
     <property name="minimum">
      <double>-340282346638528859811704183484516925440.000000000000000</double>
     </property>
     <property name="maximum">
      <double>340282346638528859811704183484516925440.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="bouncinessLabel">
     <property name="text">
      <string>Bounciness</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="ValuePicker" name="bounciness">
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.050000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="velocityLabel">
     <property name="text">
      <string>Velocity</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="velocityLayout">
     <item>
      <widget class="ValuePicker" name="velocityX">
       <property name="minimum">
        <double>-340282346638528859811704183484516925440.000000000000000</double>
       </property>
       <property name="maximum">
        <double>340282346638528859811704183484516925440.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ValuePicker" name="velocityY">
       <property name="minimum">
        <double>-340282346638528859811704183484516925440.000000000000000</double>
       </property>
       <property name="maximum">
        <double>340282346638528859811704183484516925440.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ValuePicker" name="velocityZ">
       <property name="minimum">
        <double>-340282346638528859811704183484516925440.000000000000000</double>
       </property>
       <property name="maximum">
        <double>340282346638528859811704183484516925440.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComponentGroup&lt;Raz::RigidBody&gt;</class>
   <extends>QGroupBox</extends>
   <header>RaZor/Interface/ComponentGroup.hpp</header>
  </customwidget>
  <customwidget>
   <class>ValuePicker</class>
   <extends>QDoubleSpinBox</extends>
   <header>RaZor/Interface/ValuePicker.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
